bp_common/software/py/nbf.py

```diff
@@ -135,7 +135,8 @@ def init_dram(self):
     for k in sorted(self.dram_data.keys()):
       addr = k
       opcode = self.get_opcode(addr)
-      self.print_nbf(opcode, addr, self.dram_data[k])
+      if self.dram_data[k] != 0:
```

Comment:
Can we make this a parameter in the nbf loader?  For example, on FPGA, zero-ing out is necessary